test_python/layers/test_avgpool_autotune.py

```diff
@@ -31,7 +31,7 @@ def test_avgpool_autotune(self):
         # NOTE: take note of use of {{ }} below for handling TC with scalars
         LANG = """
         def avgpool(float(B, C, H, W) input) -> (output) {{
-            output(b, c, h, w) += input(b, c, h * {sH} + kh, w * {sW} + kw) where kh in 0:{kH}, kw in 0:{kW}
```

Comment:
I think we should have a warning for such uninitialized reductions if the target tensor is not input.

Reply:
agreed. Would you like to create an issue for it? I believe there was a discussion on this but there was no resolution IIRC.

Reply:
#190